custom_config.py

```diff
@@ -54,13 +56,26 @@ def _parse_user_is_inactive_relative_time(self, key):
             weeks=+self.user_is_inactive[key].get("weeks", 0),
         )
 
+    def get_allowed_ports(self, test_id):
+        return self.get_allowed_ports_from_test_id(test_id) | self.allowed_ports_global
```

Comment:
so the readme config allows port 25 to the bastion?

Reply:
yes